spec/spec_helper.rb

```diff
@@ -11,6 +11,9 @@
 RSpec.configure do |config|
   SimpleCov.start
 
+  config.filter_run focus: true
+  config.run_all_when_everything_filtered = true
```

Comment:
Just making rspec smarter so it respects focused specs.  Makes guard much nicer to use.